libs/core/src/components/pds-accordion/pds-accordion.tsx

```diff
@@ -24,33 +23,23 @@ export class PdsAccordion {
    * Can be used to manually set the open state of the accordion.
    * @defaultValue false
    */
-  @Prop({
-    attribute: 'open',
-    mutable: true,
-    reflect: true
-  }) isOpen: boolean = false;
+  @Prop({ attribute: 'open', mutable: true, reflect: true }) isOpen: boolean = false;
```

Comment:
Minor suggestion, but legibility was better the other way around when on multiple lines.

Reply:
@ju-Skinner I agree, I think I need to adjust some linting settings in VS Code because it was complaining about it not being on one line.